test/src/test/java/org/corfudb/integration/cluster/Harness/Node.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.integration.cluster.Harness;
+
+import lombok.Getter;
+
+/**
+ * An abstraction for a corfu server/node.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+public class Node {
+
+    @Getter
+    final String address;
+
+    @Getter
+    final String clusterAddress;
```

Comment:
Does this clusterAddress refer to the clusterId referenced in the layout? If this is the case could we keep the same naming for consistency?

Reply:
its the cluster connection string (i.e. "addr1:port1,addr2:port2")